CHANGELOG.md

```diff
@@ -151,6 +161,8 @@ classes in `:partiql-ast` and `:partiql-plan`.
   - `org.partiql.lang.errors.ProblemSeverity` -> `org.partiql.errors.ProblemSeverity`
   - `org.partiql.lang.errors.ProblemHandler` -> `org.partiql.errors.ProblemHandler`
 - **Breaking** the `sourceLocation` field of `org.partiql.errors.Problem` was changed from `org.partiql.lang.ast.SoureceLocationMeta` to `org.partiql.errors.ProblemLocation`.
+- Removed `Nullable<Value` implementations of PartiQLValue and made the standard implementations nullable.
+- Using PartiQLValueType requires optin; this was a miss from an earlier commit.
```

Comment:
Was this meant to be added to `0.13.2-alpha`?

Reply:
0.13.0; The changelog looks correct to me.